transformer/transformer.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, process_cmd, d_model, dropout_rate):
                 self.functors.append(
                     self.add_sublayer(
                         "layer_norm_%d" % len(
-                            self.sublayers(include_sublayers=False)),
+                            self.sublayers(include_self=True)),
```

Comment:
include_sublayers=False is a wrong usage and this should be removed instead of using include_self